libwallet/assets/btc/sync.go

```diff
@@ -222,114 +217,93 @@ func (asset *Asset) handleSyncUIUpdate() {
 }
 
 func (asset *Asset) handleNotifications() {
-	t := time.NewTicker(syncIntervalGap)
+	t := time.NewTicker(time.Second)
+	defer t.Stop()
+	notes := asset.Internal().BTC.NtfnServer.TransactionNotifications()
```

Comment:
notes is only transaction notifications not all notification for syncing progress, so the progress bar can't update percent for users

Reply:
I think it can if we just ask btcwallet what its best block is.

We **can't** use the notifications from neutrino is the thing.